STEP_Project/tests/OUListContentTest.html

```diff
@@ -22,7 +22,7 @@
                 <td>Row3</td>
             </tr>	
         </tbody>
-    </table>
+    </table> -->
```

Comment:
Optional: If this is meant to be there only for the sake of representing dynamically created DOM we should delete it. Each test will anyway render the table in the DOM that should be enough for debugging purposes.

Reply:
deleted this table.